scripts/rspec/rspec-templates/Test.CS.cs

```diff
@@ -25,8 +25,10 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class $DiagnosticClassName$Test
     {
+        private readonly VerifierBuilder builder = new VerifierBuilder<$DiagnosticClassName$>();
```

Comment:
This is the main change, so all new rules follows this pattern even for a single method